setup.py

```diff
@@ -1,17 +1,25 @@
 from setuptools import setup
 
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+except(IOError, ImportError):
+    long_description = open('README.md').read()
+
```

Comment:
I think I agree with you that PyPI won't render the markdown in to HTML as it will with rST, so I agree with you there.

Stripping the `\r` is supposed to "fix the issue where only the first line of your README appears in pypi". Using `io.open()` handles setting the encoding, but it probably doesn't matter here.